opm/common/OpmLog/EclipsePRTLog.hpp

```diff
@@ -29,16 +29,21 @@ namespace Opm {
 class EclipsePRTLog : public StreamLog {
 
 public:
-    using StreamLog::StreamLog;
 
     void addTaggedMessage(int64_t messageType, const std::string& messageTag, const std::string& message);
 
     size_t numMessages(int64_t messageType) const;
 
     ~EclipsePRTLog();
 
+    EclipsePRTLog(const std::string& logFile , int64_t messageMask,
```

Comment:
> If so, that's nothing more than initialising the base portion of the derived object. Even C++98, long before delegating constructors, requires that this be done.

Unless you wanted the default constructor obviously, but yes.

> There is no using in the patch, but there is in the current master (and the current state of the PR does not remove it).

Right, that explains the confusion.